api-standards/structure.md

```diff
@@ -0,0 +1,84 @@
+An extension of the [API Standards](../API_STANDARDS.md), with implementation details specific to development of RESTful APIs for ONS.
+
+## RESTful API Structure and Syntax
+
+### Syntax & types
+
+  * Field names snake case, e.g. `total_count`
+  * Use appropriate types for fields, e.g. numbers should be ints not strings
+  * Timestamps should be UTC (optionally, with a timezone)
+  * Root endpoint IDs should always be referred to as `id`, rather than name-spaced. All IDs under this level will need descriptive placeholder names. e.g. /datasets/<id>/editions/<edition> where /datasets is the root endpoint
+
+### Methods & Behaviours
+ * Use the `GET` method for search endpoints
+ * `POST` endpoints should typically generate IDs, and these should use GUIDs not auto-incrementing counts
```

Comment:
POST endpoints should return tag

Reply:
Do you mean Etag? I can do this in my next PR

Reply:
An etag? I wasnt sure that standard had been approved yet?I know Nathan plans to document etags so I might leave that for him unless you mean something else